backend/tests/integration-async/records/indexer/test_literature.py

```diff
@@ -270,7 +277,17 @@ def test_record_created_through_api_is_indexed(inspire_app, clean_celery_session
         "/api/literature", json=data, headers=headers, content_type=content_type
     )
     assert response.status_code == 201
-    assert_es_hits_count(1)
+
+    def assert_record_in_es():
+        results = (
+            LiteratureSearch()
+            .query_from_iq(f"recid:{data['control_number']}")
+            .execute()
+            .hits
+        )
+        assert len(results) == 1
```

Comment:
I would just assert if results

Reply:
or can't we used the function you have created instead of duplicating it?

Reply:
yes, i've just done it